string src
string alt
---
bool success